cmake/SetupRajaOptions.cmake

```diff
@@ -17,8 +17,6 @@ option(RAJA_ENABLE_ROCTX "Build with ENABLE_ROCTX support" Off)
 option(RAJA_ENABLE_TBB "Build TBB support" Off)
 option(RAJA_ENABLE_TARGET_OPENMP "Build OpenMP on target device support" Off)
 option(RAJA_ENABLE_SYCL "Build SYCL support" Off)
-option(RAJA_ENABLE_EXTERNAL_CUB "Use an external cub" Off)
```

Comment:
@davidbeckingsale the changes to this file are mostly what I was concerned about. You're good with these changes

Reply:
Yes, due to ordering they had to be moved from here, and the changes @MrBurmark made still allow for the user to override any of the new config.